### PR TITLE
Draft release notes

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: doc/release-notes.md
Comment (chromatic) [inline doc/release-notes.md:67]: Typo: import

**Review by pradip-pack — APPROVED** [review]

**Comment by ReverseControl** [comment]
What is the difference between "recommended fees" and the "default fees"? I am trying to make sense of the following 1.14.4 Release Notes: 

```
The default fee used when deciding if a transaction should be accepted 
from a node is 0.001 DOGE (reduced from 1 DOGE).

These will not be immediately user visible, as the recommended fees do not change,
in order to retain backwards compatibilty while the network upgrades. We will release
1.14.5 to reduce the recommended fees, shortly, or you can run Dogecoin Core
with -paytxfee=0.001 to reduce your default fees manually.
```

**Comment by patricklodder** [comment]
Need this back to draft because not all the described features are in yet. I'll have changes depending on what finally makes it in.

**Comment by ReverseControl** [comment]
@patricklodder 

What's the deadline for the 1.14.4 release?

**Comment by patricklodder** [comment]
> What's the deadline for the 1.14.4 release?

I was hoping that we could close it out yesterday, but it appears that we'll need some time during the weekend to finalize open items. I still see #2297 and #2441 as important to make it in if we can. I'd also like to have a definite test on the fee interoperability, am still working on that today because I've been postponing it in favor of reviews on quick-wins the last couple of days.

**Review by michilumin — DISMISSED** [review]
glad these release notes cover the two-stage fee release

**Comment by patricklodder** [comment]
I have my edits PR'd in https://github.com/rnicoll/dogecoin/pull/28 for you.

**Comment by patricklodder** [comment]
One note aside from the proposed edits, just for consideration: I'm not sure about the promotion of personal twitter accounts in release notes. It promotes and sets an air of importance around people with commit access whereas it doesn't do that for people in the list of contributors, even though some may have worked very hard on this release. Perhaps, this is better left out.

**Comment by michilumin** [comment]
no strong opinion on the twitter thing. twitter does seem a bit transient...

**Comment by rnicoll** [comment]
> One note aside from the proposed edits, just for consideration: I'm not sure about the promotion of personal twitter accounts in release notes. It promotes and sets an air of importance around people with commit access whereas it doesn't do that for people in the list of contributors, even though some may have worked very hard on this release. Perhaps, this is better left out.

Yeah I did blink slightly when I saw them, but figured if they were in I would keep it consistent. Removed them now.

**Review by michilumin — APPROVED** [review]
second review, looks ok, covers everything i can think of.

**Review by macotouenaca — COMMENTED** [review]
ビックマネ

**Review by macotouenaca — COMMENTED** [review]
ビックマネ

**Pull request merged**
